common/evaluate.py

```diff
@@ -1,18 +1,15 @@
-from .evaluators.sst_evaluator import SSTEvaluator
-from .evaluators.reuters_evaluator import ReutersEvaluator
+from common.evaluators.classification_evaluator import ClassificationEvaluator
 
 
 class EvaluatorFactory(object):
     """
     Get the corresponding Evaluator class for a particular dataset.
     """
     evaluator_map = {
-        'SST-1': SSTEvaluator,
-        'SST-2': SSTEvaluator,
```

Comment:
So we have decided to remove SST's altogether?

Reply:
@achyudh given that the current changes aren't causing any performance change, shall I merge the PR? CC: @daemon 

I find this PR okay!

Reply:
Yeah, we decided to remove SST and add it back in later. I checked for performance changes, didn't find any when I ran LSTM Reg and KimCNN for 3 epochs.